python/lsst/ts/m2/csc.py

```diff
@@ -156,18 +157,16 @@ async def set_mount_elevation_callback(self, data: salobj.BaseMsgType) -> None:
         """
 
         if self.controller_cell.are_clients_connected():
-            await self.controller_cell.client_telemetry.write(
-                MsgType.Telemetry,
-                "elevation",
-                msg_details=dict(actualPosition=data.actualPosition),
-                comp_name="MTMount",
-            )
+            await self.controller_cell.set_external_elevation_angle(data.actualPosition)
 
     async def set_mount_elevation_in_position_callback(
         self, data: salobj.BaseMsgType
     ) -> None:
         """Callback function to notify the mount elevation in position.
 
+        Note. This function will be removed after the CSC communicates with
```

Comment:
Consider using a [deprecation warning](https://numpydoc.readthedocs.io/en/latest/format.html#deprecation-warning) instead.

Basically change this to:


```
.. deprecated:: 0.8.0
          This function will be removed after the CSC communicates with the cRIO directly.
```

at the end of the docstring.

Reply:
Updated in the commit: [71466d9](https://github.com/lsst-ts/ts_m2/pull/45/commits/71466d9802e0a0c6b2ae58e37845e0cd3430fc94). Thanks!